web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -0,0 +1,67 @@
+# PDEP-10: PyArrow as a required dependency
+
+- Created: 17 April 2023
+- Status: Under discussion
+- Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
+              [#52509](https://github.com/pandas-dev/pandas/issues/52509)
+- Author: [Matthew Roeschke](https://github.com/mroeschke)
+- Revision: 1
+
+## Abstract
+
+This PDEP proposes that:
+
+- PyArrow becomes a runtime dependency starting pandas 2.1
+- The minimum version of PyArrow supported starting pandas 2.1 is version 7.
```

Comment:
Would this version be consistent across the entire pandas API?

e.g. If I wanted to bump the pyarrow version for just the CSV parser to something higher, would I be able to do it?

Reply:
The minimum version would be consistent across the library, but IMO that shouldn't stop development of features that exist in newer versions of pyarrow (we already do this with version checking or try/except)